src/main/java/com/conveyal/datatools/manager/jobs/feedmerge/StopsMergeLineContext.java

```diff
@@ -0,0 +1,107 @@
+package com.conveyal.datatools.manager.jobs.feedmerge;
+
+import com.conveyal.datatools.manager.jobs.MergeFeedsJob;
+import com.conveyal.gtfs.error.NewGTFSError;
+import com.conveyal.gtfs.loader.Table;
+import com.csvreader.CsvReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Set;
+import java.util.zip.ZipOutputStream;
+
+import static com.conveyal.datatools.manager.jobs.feedmerge.MergeFeedsType.SERVICE_PERIOD;
+import static com.conveyal.datatools.manager.utils.MergeFeedUtils.stopCodeFailureMessage;
+
+public class StopsMergeLineContext extends MergeLineContext {
+    private static final Logger LOG = LoggerFactory.getLogger(StopsMergeLineContext.class);
+
+    private boolean stopCodeMissingFromFutureFeed = false;
+
+    public StopsMergeLineContext(MergeFeedsJob job, Table table, ZipOutputStream out) throws IOException {
+        super(job, table, out);
+    }
+
+    @Override
+    public void checkFirstLineConditions() throws IOException {
+        checkStopCodeStuff();
+    }
+
+    @Override
+    public void checkFieldsForMergeConflicts(Set<NewGTFSError> idErrors) throws IOException {
+        checkRoutesAndStopsIds(idErrors);
+    }
+
+    private void checkStopCodeStuff() throws IOException {
```

Comment:
This could have a better name, or a comment explaining what it does.

Reply:
Renamed with JavaDoc in https://github.com/ibi-group/datatools-server/pull/376/commits/06b2156db917fa518ba09f157a55369277cc7aa7.